apps/server/src/stores/token-player-map.ts

```diff
@@ -1,19 +1,35 @@
 import { AuthToken, PlayerId, socketId } from '@razor/models';
 
 interface MapData {
-  playerId: PlayerId;
+  playerId?: PlayerId;
   socketId: socketId;
 }
 
+interface mapAsObject {
+  [key: string]: MapData;
+}
+
 class TokenPlayerMap {
   private map: Map<AuthToken, MapData> = new Map<PlayerId, MapData>();
 
-  addPlayer(
-    authToken: AuthToken,
-    playerId: PlayerId,
-    socketId: socketId,
-  ): void {
-    this.map.set(authToken, { playerId, socketId });
+  /** Create new map entry with auth token and socket Id.
+   * @param authToken - auth token of the player
+   * @param socketId - socket id of the player
+   */
+  addSocketId(authToken: AuthToken, socketId: socketId): void {
+    this.map.set(authToken, { socketId });
+  }
+
+  /** Add player id to the existing map entry.
+   * @param socketId - socket id of the player
+   * @param playerId - player id of the player
+   */
+  addPlayerId(socketId: socketId, playerId: PlayerId): void {
+    for (const [key, value] of this.map.entries()) {
+      if (value.socketId === socketId) {
+        this.map.set(key, { ...value, playerId });
+      }
+    }
   }
 
   getPlayer(authToken: AuthToken): MapData | null {
```

Comment:
missing comments on other methods

Reply:
Main has comments for other methods
https://github.com/the-ai-team/razor/pull/57/files